github/resource_github_codespaces_secret.go

```diff
@@ -0,0 +1,182 @@
+package github
+
+import (
+	"context"
+	"encoding/base64"
+	"log"
+	"net/http"
+
+	"github.com/google/go-github/v53/github"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceGithubCodespacesSecret() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGithubCodespacesSecretCreateOrUpdate,
+		Read:   resourceGithubCodespacesSecretRead,
+		Update: resourceGithubCodespacesSecretCreateOrUpdate,
+		Delete: resourceGithubCodespacesSecretDelete,
+
```

Comment:
Why does this resource not support importing?

Reply:
It seems to not be implemented on the `resourceGithubActionsSecret` or `resourceGithubDependabotSecret` aswel, though org secrets do have it. Should i open an issue to do all 3?

Reply:
@KenSpur that would be excellent, thanks! In an ideal world, each resource should support importing.